### ISSUE TITLE
Building an Artificial Neural Network with TensorFlow 2

### ISSUE THREAD

**Issue by Inyrkz** [section-engineering-education/engineering-education#2390]
We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner. Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for articles to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles to go over.

**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

*We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.*

### Proposed title of the article
Your title should be descriptive of the article/tutorial. Be Specific. *Use keyword research to gain your article a higher ranking.*

Building an Artificial Neural Network with TensorFlow 2

### Introduction paragraph (2-3 paragraphs):
Please write the Introductory paragraph(s), that would be included in your article. We will use this writing snippet to help us assess overall quality before approval. We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.
Take your time and write the content as you would intend to get it published. 

In this guide, we will learn how to build an Artificial Neural Network (ANN) with TensorFlow 2.
One application of machine learning in business is predicting customers that will churn. If we know the customers that will churn, we can provide special services to them. We do not want to lose any customers.
We will build and train an Artificial Neural Network to predict customers that will churn from a bank using the customer-churning dataset from Kaggle. Some features of the dataset we will use are credit score, country, gender, age, tenure, balance, etc. 
After reading this guide, you will be able to build an ANN to solve binary classification problems with any dataset.

### Key takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article? Use this area to get your ideas down on the bulk of your article or tutorial.

- Label Encoding & One-Hot Encoding Categorical Variables,
- Build and visualize model architecture with Keras visualizer,
- Compile and train an ANN with Keras on top of TensorFlow 2,
- Make Predictions on a new customer,
- Check the model's accuracy score and create a confusion matrix.

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.

- https://keras.io/api/

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by hectorkambow** [comment]
seems like a helpful topic - lets please be sure it add value beyond what is in the official docs and that it does not overlap with any existing EngEd articles or incoming topic suggestions (if you haven't already). - approved @Inyrkz
